Fit function

AReyH opened the pull request:
Added function code after review

AReyH committed:
Added fit.py reluctantly

AReyH committed:
Added fit.py reluctantly again

AReyH committed:
Added __name__ == __main__

s-voon commented on src/autopredictor/fit.py:
```diff
@@ -22,3 +31,49 @@ def fit(X_train,X_test,y_train,y_test):
     dict 
         A dictionary containing the performance scores for each model and metric.
     """
```
Please include an example here since we implemented our code this time

s-voon reviewed (changes requested):
Hey Arturo! The function looks good in general. Just a minor comment to include examples in the docstrings too now that we have to code ready. Also just a small personal preference thing, the spacing in between code is inconsistent. One that stood out is the metrics dictionary (line 63-64), It would be nice to remove that. Thankiee.

AReyH committed:
Test

AReyH committed:
Added tests

AReyH committed:
Corrected tests

AReyH committed:
Added examples to docstring

s-voon commented on tests/test_fit.py:
```diff
@@ -0,0 +1,53 @@
+
+
+
+#current_dir = os.path.dirname(os.path.abspath(__file__))
```
This part is commented out. Do you wanna remove it if its not needed? Is there a reason why you would wanna leave it in?

s-voon reviewed (changes requested):
Well done~ Minor comment on the test_fit.py but otherwise looks good to me!

s-voon commented on tests/test_fit.py:
```diff
@@ -0,0 +1,53 @@
+X_train, X_test, y_train, y_test = train_test_split(X, y)
+
+
+def test_output_dict():
```
Also can you please add like a short docstring for all the functions please. Thankieee~

AReyH committed:
Update test_fit.py

AReyH committed:
Update test_fit.py

Deleted some unnecessary code from the test_output_dict() function and added docstrings

AReyH committed:
Update test_fit.py

Deleted if __name__ == '__main__': from test

s-voon reviewed (approved):
Looks good!
